### ISSUE TITLE
[BUG/ISSUE] Cannot update applications. Updates not parsed correctly.

### ISSUE THREAD

**Issue by adripo** [marticliment/WingetUI#930]
## Before submitting the issue

- [ ] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [ ] **I have checked that I am running the latest version of WingetUI**
- [ ] **I have tried resetting and reinstalling WingetUI and the issue persisted or happened again**
- [ ] **I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions**
- [ ] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#issues) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**

## Describe the bug
The updates are not shown correctly and some packages are not upgradeable. The issue could be generated by the `The following packages have an upgrade available, but require explicit targeting for upgrade:` text present when executing `winget upgrade`

## Screenshots
![image](https://user-images.githubusercontent.com/26493496/232934715-c6fa2f81-6022-4902-a863-6a1b157bcf6d.png)

## The Log
```
winget.exe upgrade
Name                                  Id                              Version       Available     Source
--------------------------------------------------------------------------------------------------------
Malwarebytes Windows Firewall Control BiniSoft.WindowsFirewallControl 6.9.0.0       6.9.2.0       winget
Node.js                               OpenJS.NodeJS.LTS               18.15.0       18.16.0       winget
3 upgrades available.

The following packages have an upgrade available, but require explicit targeting for upgrade:
Name    Id              Version  Available Source
-------------------------------------------------
Discord Discord.Discord 1.0.9011 1.0.9012  winget
```

**Comment by marticliment** [comment]
This issue has been fixed previously, but the fix hasn't been released yet

**Comment by Apreche** [comment]
Has this fix been released yet? It's over a month, and this is still happening to me. It's a pretty huge bug, so I would think it would be high priority.

**Comment by marticliment** [comment]
did you try the latest betas?

**Comment by Apreche** [comment]
> did you try the latest betas?

Oh, I have to use beta? I've only ever used the stable version.

**Comment by adripo** [comment]
@Apreche for me the issue was solved with version [1.7.1](https://github.com/marticliment/WingetUI/releases/tag/1.7.1)
